system/modules/isotope/library/Isotope/Interfaces/IsotopeProductCollection.php

```diff
@@ -16,6 +16,7 @@
 use Isotope\Model\Config;
 use Isotope\Model\ProductCollectionItem;
 use Isotope\Model\ProductCollectionSurcharge;
+use Isotope\Template;
```

Comment:
This should be `Contao\Template`, it does not have to be an Isotope-specific one.

Reply:
Done